vhost_user_block/src/lib.rs

```diff
@@ -232,10 +272,38 @@ impl VhostUserBackend for VhostUserBlkBackend {
         match device_event {
             q if device_event < self.config.num_queues => {
                 let mut vring = vrings[q as usize].write().unwrap();
-                if self.process_queue(&mut vring) {
-                    debug!("signalling queue");
-                    vring.signal_used_queue().unwrap();
+
+                if self.poll_queue {
+                    // Actively poll the queue until POLL_QUEUE_US has passed
+                    // without seeing a new request.
+                    let mut now = Instant::now();
+                    loop {
```

Comment:
All the events come here are from the epoll in vhost_user_backend, is it necessary to add a loop here?

Reply:
This is for polling the queue for new elements without having to wait for an ioeventfd signal.